### DISCUSSION TITLE
InvalidArgument desc = etcdserver: user name is empty

### DISCUSSION THREAD

**Discussion by shaiktas** [milvus-io/milvus#26923] [Q&A and General discussion]
Hello,
Greetings for the Day. I have deployed all required third party dependencies etcd, pulsar, mysql and kafka. all are healthy. When I start milvus micro services post third party dependencies, all micro services are unhealthy. Up on checking logs below Warnings/Error is exist in all micro services (root coordinator, query coordination, index coordinator, proxy...). Please help how this can be resolved:
-----------------------

{"level":"warn","ts":"2023-09-07T18:09:25.642Z","logger":"etcd-client","caller":"v3@v3.5.5/retry_interceptor.go:62","msg":"retrying of unary invoker failed","target":"etcd-endpoints://0xc0005a9880/milvus-svc-tar-etcd:2379","attempt":0,"error":"rpc error: code = InvalidArgument desc = etcdserver: user name is empty"}

**Comment by yhmo** [comment]
"etcdserver: user name is empty" ------ seems this error is for etcd auth, but Milvus doesn't support auth etcd.

**Reply by shaiktas** [reply]
Thanks for the hint @yhmo . After enabling etcd without auth, it solves the issue. But in production env is it recommended to use etcd without authentication? Do we have any Milvus releases/plans to support with authentication based etcd?

**Reply by yhmo** [reply]
This enhancement is not in our plan since we didn't receive the requirement for etcd auth. To get more attention, you can create a "feature request" issue and write the requirement in the "[what feature you want](https://github.com/milvus-io/milvus/discussions/6302)" page. Then we can arrange our plan.
